### DISCUSSION TITLE
DEX implementation?

### DISCUSSION THREAD

**Discussion by dorkmo** [Chia-Network/chia-blockchain#866] [General]
Whats the game plan for reaching out to groups like [bisq](https://github.com/bisq-network/bisq)? Should we start an issue on their github to add chia?

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note there are a number of DEXs that support xch and CATs:
https://www.chia.net/buy-xch/#load-your-wallet

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
